src/main/java/com/movie_collection/dal/dao/MovieDAO.java

```diff
@@ -1,8 +1,9 @@
 package com.movie_collection.dal.dao;
 
+import com.movie_collection.be.Category;
```

Comment:
maybe can we stick with some consistency and have all our Dao and method inside the classes in some better order ? so that it does not look we have been just adding method however we wanted and whenever 
I have in mind something like its follow (getAllWhatever,getAllWhatEverByEmail,getAllByName, and then have all create then all update methods  and lastly have all the methods to take care of the delegation ?

Reply:
This way seems fine